about-us/what-we-do.md

```diff
@@ -2,15 +2,14 @@
 
 We work on:
 
-* Solidus/Spree e-commerce design and development;
-* design and development of web applications;
-* design and development of CMS sites;
-* development of static sites;
-* web branding;
-* management of PPC, SEO, social media campaigns and newsletters.
+* Solidus eCommerce design and development;
+* Web application design;
+* Ruby on Rails development;
+* Vue.js/React/Angular development;
```

Comment:
I think we should remove Angular

Reply:
@kennyadsl yeah I was not entirely sure, removing.